utils/grid_visualizer/tile_colormap.json

```diff
@@ -0,0 +1,75 @@
+[
```

Comment:
I think we should end up encoding this data in the `sim.v` and/or `pb_type.xml` as `<metadata>` tags?

Reply:
That's a good idea. I remember that we had a conversation about that. Maybe an issue is already posted.

Reply:
Do you want to create / find such a issue?

Reply:
Couldn't find it so I added a new one: https://github.com/SymbiFlow/symbiflow-arch-defs/issues/741